src/playground-code-editor.ts

```diff
@@ -380,6 +380,13 @@ export class PlaygroundCodeEditor extends LitElement {
       case 'ts':
         return 'google-typescript';
       case 'js':
+      case 'json':
+        // While the stock CodeMirror JavaScript mode has a restricted "json"
+        // mode, the google-javascript mode does not (which we use because it
+        // supports html-in-js highlighting). Adding the CodeMirror JavaScript
+        // mode would add ~50KiB minified + brotli, so let's just put up with
+        // the fact that you'll get highlighting for JS even though it's not
+        // valid JSON.
```

Comment:
sidenote, do we have an idea on current bundle size?

Reply:
terser + brotli:

service worker: 1.74 KiB
typescript worker: 525.28 KiB
ide: 114.72 KiB